### PR TITLE
SJE-01

### PR DESCRIPTION
**Pull request by Bubblegumi**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,15 +6,76 @@
...
+                                <li class="menu__item"><a href="#" class="menu__link">Contact us</a></li>
+                            </ul>
+                        </div>
+                        <div class="col-md-3 d-flex justify-content-sm-center justify-content-md-star">
+                            <div class="profile d-flex">
+                                <div class="profile__item d-flex align-items-center">
+                                    <img src="./icons/location.png" class="profile__icon" alt="">
+                                    <a href="#" class="profile__link">Location</a>
+                                    <select name="location" id="">
+                                        <option value="zalupa"></option>
```
Comment (mosiichuk) [inline src/index.html:31]: Please remove this.

File: src/styles/style.sass [hunk 2]
```diff
@@ -29,20 +29,134 @@ body
...
+
+    & .film-first
+        background: url('../img/movie1.png') center /cover
+        width: 100%
+        height: 389px
+        
+        &:hover
+            background: linear-gradient(0deg, rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0.2))
+        
+
```
Comment (mosiichuk) [inline src/styles/style.sass:106]: Remove extra line.

File: src/index.html [hunk 3]
```diff
@@ -6,15 +6,76 @@
...
+                                    <img src="./icons/profile.png" class="profile__icon" alt="">
+                                    <a href="#" class="profile__link">Log in</a>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <div class="film-anons d-flex">
```
Comment (mosiichuk) [inline src/index.html:46]: There is no word `anons`. Please rename.

File: src/index.html [hunk 4]
```diff
@@ -6,15 +6,76 @@
...
+            <div class="film-first"><div class="film-anons__desc">
+                <p class="film-anons__text">Fantacy/ Action /Cyberpank</p>
+                <h2 class="film-anons__title">Blade Rubber 2049</h2>
+            </div>
+        </div>
```
Comment (mosiichuk) [inline src/index.html:51]: Fix indentation for element above.

Comment (mosiichuk) [reply src/index.html:51]: Add description for all movies please.

File: src/styles/style.sass [hunk 5]
```diff
@@ -155,11 +166,105 @@ a
...
+        content: ''
+        height: 2px
+        width: 100%
+        background: $accent
+        position: absolute
+        left: 0
+        bottom: 20%
+
+// log in
+
```
Comment (Bubblegumi) [inline src/styles/style.sass:267]: attempt to make login section

**Pull request merged**